implementations/python/sugar-lift-python-source/src/sugar_lift_python_source/dependency_export_adapter.py

```
@@ -448,6 +531,349 @@ def _import_coordinate(
+            # deferred.  FunctionDef.sugar() builds FunctionUniverseSugar (the
+            # call-universe value), which reduces the body as if it ran at the
+            # def site — wrong for module sequencing.  Import/Pass already use
+            # InertSugar; def/class match that door for fall-through only.
+            if isinstance(statement, (FunctionDef, AsyncFunctionDef, ClassDef)):
+                sugars.append(InertSugar(site=statement.fragment))
+            else:
+                sugars.append(statement.sugar())
+        exits = reduce_block_to_exitset(tuple(sugars))
+    except SugarNotWritten:
+        return False
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)